src/site/content/en/blog/file-handling/index.md

```diff
@@ -85,6 +85,12 @@ an object with two properties:
   icons. Some operating systems allow a file type association to display an icon that is not just
   the associated application icon, but rather a special icon related to the use of that file type
   with the application.
+- A `"launch_type"` property that defines whether multiple files should be opened in a single
+  client or in multiple clients. If omitted, this defaults to `"single-client"`. If the user
```

Comment:
```suggestion
  client or in multiple clients. The default is `"single-client"`. If the user
```

Reply:
I've never encountered or heard of anyone having trouble with wording like I'm suggesting.